macos/ONLYOFFICE/update/updates/ONLYOFFICE/changes/7.5.0/changes.html

```diff
@@ -91,46 +91,71 @@ <h2>New Features</h2>
 
         <h4>All Editors</h4>
         <ul>
-            <li>TODO</li>
+            <li>Added the support for >200% (225, 250, 275, 300, 350, 400, 450, 500 interface scaling)</li>
+            <li>In the <code>“Open Recent”</code> section, added files in the format corresponding to the editor from the “Recent files” list of the application</li>
+            <li>Updated <code>VLC</code> libraries (for Windows)</li>
+            <li>Updated the <code>“About”</code> page and the “Check for updates” block (for Windows)</li>
+            <li>Added a number of changes for the deb and rpm packages</li>
+            <li> scaling is preserved for future sessions</li>
+            <li>Add a menu for quick access to the most popular symbols to the button for inserting symbols</li>
+            <li>Support for <code>SVG</code></li>
+            <li>Add a setting to hide the toolbar for equation</li>
+            <li>Add hints for images in SmartArt objects</li>
+            <li>Add the ability to open files protected with a password in the embedded viewer</li>
+            <li>Add the ability to edit points of the autoshape border to the right panel</li>
+            <li>Add the support for 225 and 275 interface scaling</li>
+            <li>Only <code>Ctrl + .</code>/<code>Ctrl + ,</code> keyboard shortcuts are now used for superscript/subscript characters</li>
+            <li>Change keyboard shortcuts for moving through the text on macOS</li>
         </ul>
         <h4>Document Editor</h4>
         <ul>
-            <li>TODO</li>
+            <li>Automatic hyphenation</li>
+            <li>Change the selection logic, add the <code>“Smart paragraph selection”</code> setting</li>
+            <li>Add the <code>“Remove content control when contents are edited”</code> setting</li>
+            <li>Change behavior of the <code>Accept/Reject</code> review buttons in the toolbar</li>
+            <li>Disable the Zoom out action for the <code>Ctrl + Numpad</code> keyboard shortcut due to the conflict with inserting an em dash</li>
+            <li>Change the keyboard shortcut for strikeout text formatting on macOS: <code>Ctrl + 5</code> replaced with <code>Cmd + Shift + X</code></li>
+            <li>Change the keyboard shortcut for inserting ellipsis on macOS: <code>Ctrl + Option + .</code> replaced with <code>Option + ;</code></li>
         </ul>
         <h4>Spreadsheet Editor</h4>
         <ul>
-            <li>TODO</li>
+            <li>The ability to display only formulas in cells</li>
+            <li>Trace precedents / dependents</li>
+            <li>New function: <code>SORTBY</code></li>
+            <li>Margins presets for printing</li>
```

Comment:
Added presets for "Margins" and the ability to show/hide "Print titles" in the "File" > "Print" menu